devtools/chain/config.toml

```diff
@@ -55,10 +51,9 @@ metrics = true
 [rocksdb]
 max_open_files = 64
 cache_size = 134217728
-
 # Provide an options file to tune RocksDB for your workload and your system configuration.
 # More details can be found in [the official tuning guide](https://github.com/facebook/rocksdb/wiki/RocksDB-Tuning-Guide).
-options_file = "default.db-options"
+options_file = "devtools/chain/default.db-options"
```

Comment:
Directory changes need to change the code, the current code thinks here is just a file name